core/src/main/scala/org/broadinstitute/dsde/rawls/util/WorkspaceSupport.scala

```diff
@@ -74,8 +74,7 @@ trait WorkspaceSupport {
       userHasAction <- traceDBIOWithParent("userHasAction", parentSpan)(_ => DBIO.from(samDAO.userHasAction(SamResourceTypeNames.billingProject, projectName.value, SamBillingProjectActions.createWorkspace, userInfo)))
       response <- userHasAction match {
         case true => op
-        case false => DBIO.failed(new RawlsExceptionWithErrorReport(errorReport = ErrorReport(StatusCodes.Forbidden, s"You are not authorized to create a workspace in billing project ${workspaceRequest.toWorkspaceName.namespace}")))
-      }
+        case false => DBIO.failed(new RawlsExceptionWithErrorReport(errorReport = ErrorReport(StatusCodes.Forbidden, s"You are not authorized to create a workspace in billing project ${workspaceRequest.toWorkspaceName.namespace}")))      }
```

Comment:
Not sure what's different about this line. I did have to remove some code from `develop` here when merging that checked the `CreationStatus` of the workspace's Terra billing project because we don't care about that in a PPW world

Reply:
I think the `}` moved up a line